_commands/basics/tail.md

```diff
@@ -3,22 +3,42 @@
 
 tail
 -------
-TODO: Add documentation for this command by submitting a pull request.
+`tail` is used to read the final few lines of text from a document. The default length is 10. 
 <!-- one line explanation would go here -->
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ tail example.txt
+$ tail example1.txt example2.txt
```

Comment:
what does tail do when given two commands? Add this to the useful options and explain what it does.